index.js

```diff
@@ -98,6 +100,8 @@ module.exports = {
     var options = config.options;
     var project = options.project;
 
+    console.log(options.environment);
```

Comment:
Whooops, I missed this before I merged!

Reply:
Arghs and I missed that I committed that one with the last inline comment. I'm very sorry about that. Do you want me to open a PR to remove it or are you doing it yourself?

Reply:
`index.js` is not covered by ESLint :sob:

Reply:
I've removed the `console.log` call in https://github.com/rwjblue/ember-cli-content-security-policy/commit/97cfed04ea213b9f1ac2c2f18a5c3efec67a730f (hope that was okay @rwjblue)

Reply:
@sandstrom Thanks a lot. I'm really sorry for the churn.

Reply:
@jelhan No worries, thanks for making this code better! 😄